### PR TITLE
Update resume.md (Victor)

### PR DESCRIPTION
**Pull request by spoiledhua**
Request for review @alextakahashi , thanks in advance!

- What do you think about leaving "Incoming..." to specify internships I'll be completing in the future? My thought here is that as a senior recruiting for mostly FT roles, I should demonstrate experiences I'll have by the time I begin my FT role after graduation, but it could also potentially waste space where I could be including more bullets about past experiences.

- Should I limit my Awards section? Currently I have six but am not sure if they should all be included / have the same weight.

- I try to keep my explanations for my projects relatively short as they're self-explanatory and can be viewed directly via a URL. What do you think about this approach?

### CODE REVIEW

File: resume.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+* **Libraries/Frameworks**: Pandas, TensorFlow, React
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Incoming Software Engineering Intern**: Optiver | Chicago, IL *Jun 2021 - Aug 2021*
```
Comment (alextakahashi) [inline resume.md:18]: fine to put incoming - although if you do know the projects you'll be working on that can be helpful

File: resume.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Incoming Software Engineering Intern**: Postmates X | San Francisco, CA *Mar 2021 - May 2021*
+
+**Software Engineering Intern**: NASA Jet Propulsion Laboratory | Pasadena, CA *Sep 2020 - Present*
+* Developing a vehicle simulation framework on the Mission Control team to scale support to dozens of robotic vehicles
```
Comment (alextakahashi) [inline resume.md:23]: if you can be specific about the technology you used that could be helpful, mind whatever NDA you have

File: resume.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+
+**Software Engineering Intern**: Google | Mountain View, CA *May 2020 - Aug 2020*
+* Increased Google Cloud query and data generation speed 10x by implementing a Markov chain-based tool in Java
+* Designed and scoped a project with open-ended specs by surveying engineers and iterating on design documentation
+* Generalized tokenization in several SQL dialects to increase new feature integration efficiency by 50%
+
+**Chief Operating Officer**: Illumna | Fort Lee, NJ *Oct 2017 - Oct 2019*
+* Led a team of 40+ college students to launch a nonprofit dedicated to mentoring under-resourced students
+* Increased 100+ mentees’ soft skills confidence by an average of 50% by writing 70+ pages of novel curriculum
```
Comment (alextakahashi) [inline resume.md:32]: This sounds impactful but possibly this space can be better used to talk more about your technical abilities.  Can pivot this idea to be more technical

File: resume.md [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**EasyGov**: *Goverment forms made easy* | easygov.app *May 2020 - Present*
+* Developed a platform improving access to translated government forms with 1000+ form downloads in one month
```
Comment (alextakahashi) [inline resume.md:36]: Can you support this idea with the technical implementation details?  What  tools did you use?  What are the next steps resulting from your partnerships?

**Review by alextakahashi — CHANGES_REQUESTED** [review]
TL;DR This look pretty good, strong projects + technical details to back them up.  Small nits about adding additional technical detail.  Maybe you can dive deeper into your experiences vs being broad with your projects

> What do you think about leaving "Incoming..." to specify internships I'll be completing in the future? My thought here is that as a senior recruiting for mostly FT roles, I should demonstrate experiences I'll have by the time I begin my FT role after graduation, but it could also potentially waste space where I could be including more bullets about past experiences.

I think brand recognition is a win - if you can get some details on the project youre working on that could populate the whitespace

> Should I limit my Awards section? Currently I have six but am not sure if they should all be included / have the same weight.

Depends on how much whitespace you need to populate, if you have whitespace go for it.  Two cents is that you should always prefer technical details over all else

> I try to keep my explanations for my projects relatively short as they're self-explanatory and can be viewed directly via a URL. What do you think about this approach?

your one liners sound pretty good as is

**Comment by alextakahashi** [comment]
Closing, reopen for another review
